### DISCUSSION TITLE
If “hide-default-header” removes sort functionality in columns, how do I get it back?

### DISCUSSION THREAD

**Discussion by Graphictank** [vuetifyjs/vuetify#12669] [Q&A] [answered]
I have some custom TH's, is there any way to bring back asc/desc without having to re-invent the wheel?

**Comment by rigloo** [comment]
hello, did you ever find a solution for this? I need apply some custom th but when I hide the default header it just removes sort functionality

**Comment by Graphictank** [comment, marked as answer]
Yea, for every column you have, make a long list of th's one after the other and asign them a numerical value (1, 2, 3...) sorry for the choppy code below but GH's code tag wasn't allowing me to format it properly. Hopefully this example will get you on your way. 



`th
        :key="headers[1].text"
        :class="['column sortable', pagination.descending ? 'desc' : 'asc', headers[1].value === pagination.sortBy ? 'active' : '']">
        {{ headers[2].text }}

        
        <v-btn
          :ripple="false"
          height="14"
          width="14"
          x-small
          icon
          tabindex="-1">
          <v-icon
            @click="changeSort(headers[1].value)"
            x-small
            color="grey darken-1"
            aria-label="Column Sort button">mdi-arrow-down
          </v-icon>
        </v-btn>
      </th`
____________________________________

`  methods: {

    changeSort(column) {
      if (this.pagination.sortBy === column) {
        this.pagination.descending = !this.pagination.descending;
      } else {
        this.pagination.sortBy = column;
        this.pagination.descending = false;
      }
      var order = this.pagination.descending ? "desc" : "asc";
      this.arrangedPlateList.sort(this.compareValues(column, order));
    }
  }`

**Reply by rigloo** [reply]
Thank you this saved my life

**Reply by Graphictank** [reply]
Glad it could help you out. Man, I had to go back two years to find this push. We've since moved on to SyncFusion as our grid. Way more verbose... excel-like in it's use.

**Comment by rigloo** [comment]
I appreciate it man. Vuetify has worked wonderfully for our needs. Its just
these weird specific cases that the users asked for. In this case certains
columns on a v-data-table need to be fixed to the left even when you scroll
horizontally.
